### PR TITLE
Expand readme

### PR DESCRIPTION
**Pull request by jodh-intel**

### CODE REVIEW

**Comment by jodh-intel** [comment]
Hi @iphutch - please could you review?

File: README.md [hunk 1]
```diff
@@ -5,39 +5,78 @@
 
 # runtime
 
-## Global logging
+## Introduction
 
-Additional to the (global) `--log=` option, the runtime also has the
-concept of a global logfile.
+`cc-runtime` is the next generation Intel® Clear Containers runtime.
```
Comment (devimc) [inline README.md]: `cc-runtime` is the next generation **of** Intel® Clear Containers runtime. ??

File: README.md [hunk 2]
```diff
@@ -5,39 +5,78 @@
...
-- log level
-- log message
+See [the license file](https://github.com/clearcontainers/runtime/blob/master/LICENSE) for further details.
 
-The global logfile records all log output sent to the standard logfile.
+## Hardware requirements
 
-Note that if output is disabled for the standard logfile, the global log
-will still record all logging calls the runtime makes.
+The runtime has a built-in command to determine if your host system is capable of running a Clear Container. Simply run:
```
Comment (devimc) [inline README.md]: s/a Clear Container/a Intel® Clear Container/ ?

File: README.md [hunk 3]
```diff
@@ -5,39 +5,78 @@
...
+
+## Debugging
+
+To provide a persistent log of all container activity on the system, the runtime
+provides a global logging facility. By default, this feature is disabled.
+
+To enable the global log:
+
+```bash
+$ sudo sed -i -e 's/^#\(\[runtime\]\|global_log_path =\)/\1/g' /etc/clear-containers/containers.toml
```
Comment (devimc) [inline README.md]: yeep, but what if containers.toml is not in ```/etc``` I think by default it is installed in ```/usr/local/etc```

File: README.md [hunk 4]
```diff
@@ -5,39 +5,78 @@
...
+```
+
+The path to the global log file can be determined subsequently by
+running:
+
+```bash
+$ cc-runtime cc-env | grep GlobalLogPath
+```
+
+Note that it is also possible to enable the global log by setting  the
```
Comment (devimc) [inline README.md]: double space ```setting  the```

File: README.md (hunk 1 above)
Comment (jodh-intel) [inline README.md]: Yes, I think I prefer your wording. Fixing...

File: README.md (hunk 2 above)
Comment (jodh-intel) [inline README.md]: Good catch - fixing...

File: README.md (hunk 3 above)
Comment (jodh-intel) [inline README.md]: That is handled by the note below, but yes, logically it should be handled first. Fixing...

File: README.md (hunk 4 above)
Comment (jodh-intel) [inline README.md]: Fixing...

**Comment by jodh-intel** [comment]
@devimc, @iphutch - doc updated.

File: README.md (hunk 1 above)
Comment (devimc) [inline README.md]: so is ```of``` missing ?

Comment (jodh-intel) [inline README.md]: Updated! ;)

**Comment by jodh-intel** [comment]
@devimc - could you re-review?

**Comment by devimc** [comment]
lgtm

**Pull request merged**